.pre-commit-config.yaml

```diff
@@ -22,4 +22,4 @@ repos:
     rev: 21.9b0
     hooks:
       - id: black
-        additional_dependencies: [".[python2]", "click==8.0.4"]
+        additional_dependencies: [".[python2]"]
```

Comment:
Please revert this as well

Reply:
Apologies - I am not sure I understand. Do we want click or not?

In the original file I had, click was not there...

Reply:
We want this file to be untouched by this pull request. This branch was originally made a year ago and have been changes to `rel-v7r2` since then (in this case to workaround an issue with the Python 2 compatible `black` release no longer working with newer `click` releases).

Would you like me to rebase this branch for you and squash all the commits into one?

Reply:
Yes please. In hindsight, this is the fastest!

Reply:
Done :+1:

You'll have conflicts when you try to pull locally, you can get the latest version with:

```bash
git fetch --all
# DANGER: This is destructive, make sure you're on the right branch
# and you have no local changes
git reset --hard FETCH_HEAD
```

Reply:
Thanks a lot! :+1: